runners/console/index.ts

```diff
@@ -442,10 +443,70 @@ export class Console extends Runner {
         if(!fs.existsSync(workspacesDir))
             fs.mkdirSync(workspacesDir);
         this.setVariable(this.workspaceDirectory, workspacesDir);
+     
+      return result;
+    }
 
-        return result;
+    runExecuteCommand(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0; 
+        let commandIndex;
+        if(runCommand.command.parameters[0] && runCommand.command.parameters[1]){
+            commandIndex = this.platform == ConsolePlatform.LINUX ? 1 : 0;
+        }
+        else{
+            throw new Error("You have to pass a Command for Windows and Linux based OS");
+        }
+
+        let exeCommand = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[2].args)
+        ? runCommand.command.parameters[commandIndex]+ " " +runCommand.command.parameters[2].args.join(" ")
+        : runCommand.command.parameters[commandIndex];
+
+        let dirPath = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[2].dir)
+        ? path.join(this.getVariable(this.workspaceDirectory), runCommand.command.parameters[2].dir)
+        : this.getVariable(this.workspaceDirectory)
+
+        if(runCommand.command.parameters.length > 2 && runCommand.command.parameters[2].asynchronous){
+            if(runCommand.command.parameters[3].port){
+                let process = ConsoleUtils.executeCommandAsync(exeCommand, dirPath, result,this.env);
+                if(process.pid) {
+                    this.asyncProcesses.push({ pid: process.pid, name: "ExecuteCommand", port: runCommand.command.parameters[3].port});
```

Comment:
I think we have to remove the name property here. In the killAsyncProcesses method we compare the name with the name of the process found on the given port and only kill them when the names match. So in this case the processes would never be killed